src/Hint/Context.hs

```diff
@@ -128,7 +125,7 @@ addPhantomModule mod_text =
                    (old_top, old_imps) <- runGhc getContext
```

Comment:
The comment above mentions `GHC.load`, which is no longer present in this PR's code.